### DISCUSSION TITLE
OC : Failed to load configration

### DISCUSSION THREAD

**Discussion by PriyankManiya** [macOS-Dell-Latitude-E7470/macOS-Dell-Latitude-E7470#30] [General]
My laptop configuration is
DELL latitude E-7480 (I7 - 7th gen) (Touch Screen)
512 GB M.2 SSD

I've booted Bigsure and then just copy your EFI folder according to the guide and try to install but it. is showing me this msg while loading Pendrive

**Comment by adityabakare** [comment]
What message? You need to be more specific.
Also, the boot mode should be AHCI and not RAID. Make sure you're on AHCI.

Moving this to DIscussions since this really isn't an issue.
